lib/db.py

```diff
@@ -1,94 +1,103 @@
-        cur.execute("SELECT id, subject, claim, object, statement, \"effectiveDate\", \"sourceURI\", \"howKnown\", \"dateObserved\", \"digestMultibase\", author, curator, aspect, score, stars, amt, unit, \"howMeasured\", \"intendedAudience\", \"respondAt\", confidence, \"issuerId\", \"issuerIdType\", \"claimAddress\", proof FROM \"Claim\" WHERE id > {}".format(latest_claimid))
-
-        columns = [desc[0] for desc in cur.description]
-        while True:
-            rows = cur.fetchmany()
-            if not rows:
-                break
-            for row in rows:
-                yield dict(zip(columns, row)) 
-
-def unpublished_claims_generator():
-    with get_conn().cursor() as cur:
-        # Read data from the Claim model
-        # TODO track last date and only process new claims
-        cur.execute("SELECT id, subject, claim, object, statement, \"effectiveDate\", \"sourceURI\", \"howKnown\", \"dateObserved\", \"digestMultibase\", author, curator, aspect, score, stars, amt, unit, \"howMeasured\", \"intendedAudience\", \"respondAt\", confidence, \"issuerId\", \"issuerIdType\", \"claimAddress\", proof FROM \"Claim\" WHERE \"claimAddress\" is NULL or \"claimAddress\" = ''")
-        # could refactor this section with above function
-        columns = [desc[0] for desc in cur.description]
-        while True:
-            rows = cur.fetchmany()
-            if not rows:
-                break
-            for row in rows:
-                yield dict(zip(columns, row)) 
-
+    with get_conn() as conn:
+        with conn.cursor() as cur:
+            # find latest processed claim
+            QUERY_LATEST_CLAIMID = 'SELECT MAX("claimId") FROM "Edge"'
+            cur.execute(QUERY_LATEST_CLAIMID)
+            latest_claimid = cur.fetchone()[0]
+            
+            # Read data from the Claim model
+            cur.execute("""
+                SELECT id, subject, claim, object, statement, \"effectiveDate\", 
+                \"sourceURI\", \"howKnown\", \"dateObserved\", \"digestMultibase\", 
+                author, curator, aspect, score, stars, amt, unit, \"howMeasured\", 
+                \"intendedAudience\", \"respondAt\", confidence, \"issuerId\", 
+                \"issuerIdType\", \"claimAddress\", proof 
+                FROM \"Claim\" WHERE id > %s
+            """, (latest_claimid,))
+
+            columns = [desc[0] for desc in cur.description]
+            while True:
+                rows = cur.fetchmany()
+                if not rows:
+                    break
+                for row in rows:
+                    yield dict(zip(columns, row))
 
 def execute_sql_query(query, params):
-    with get_conn().cursor() as cur:
-        cur.execute(query, params)
-        result = cur.fetchone()
-        conn.commit()
-        if result is not None:
-            col_names = [desc[0] for desc in cur.description]
-            return dict(zip(col_names, result))
-        else:
+    with get_conn() as conn:
```

Comment:
please make linter changes separately and actual changes in a smaller pr

too hard to review large pr with many lint-only changes and some real ones